code/style.css

```diff
@@ -1,3 +1,84 @@
 body {
-  background: #FFECE9;
-}
\ No newline at end of file
+  background: #F7D4BC;
+  font-family: 'Gemunu Libre', sans-serif;
+}
+
+header {
+  text-align: center;
+  font-size: 30px;
+}
+
+.profile {
+  align-items: center;
+  justify-content: center;
+  display: flex;
+  text-align:center;
+  color: #218380;
+  margin:auto;
+  width: 100%;
+  flex-direction: column;  
+}
+
+img {
+  border-radius: 50%;
+  width: 200px;
+  margin: 15px;
+  box-shadow: -30px 30px 30px rgba(0, 0, 0, 0.3);
+}
+
+p {
+  color: #AD5D4E;
+}
+
+.projects {
+  flex-direction: column;
+  display: block;
+  border-style: solid;
+  color: #AD5D4E;
+  margin: 5px;
+  text-align: center;
+}
+
+.chart-class {
+  width: 50%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
```

Comment:
The chart was very tricky to style because it doesn't really listen to whatever you style it to like other element, I also tried to style it with flex and with different align-item and justify-content etc but nothing actually happened. The only thing that seems to work is the margin: auto; that you have, the other properties could actually be removed I think. (could be worth try to remove it to have a cleaner code) I also just set a width in % and then align-self: center to make it centered and not being huge, that worked for me at least 🙂